hyperspy/io_plugins/tiff.py

```diff
@@ -382,6 +391,24 @@ def _parse_scale_unit(tiff, page, op, shape, force_read_resolution):
             if 'spacing' in imagej_metadata:
                 scales['z'] = imagej_metadata['spacing']
 
+    if ('Make' in op) and (op['Make']=="JEOL Ltd."):
```

Comment:
Slightly more readable:
```suggestion
    if op.get('Make'', None) == "JEOL Ltd.":
```

Reply:
modified.